src/containers/announcements/Announcements.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { Typography } from 'antd';
+import AnnouncementsList from '../../components/announcements-list/AnnouncementsList';
+import styled from 'styled-components';
+import { ChungusContentContainer } from '../../components';
+
+const { Title } = Typography;
+
+const Content = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const StyledTitle = styled(Title)`
+  margin: 12px;
+  margin-left: 0px;
+`;
+
+const Announcements: React.FC = () => {
+
+    React.useEffect(() => {
+        window.scrollTo(0, 0);
+    }, []);
+
+    return (
+        <>
+            <Helmet>
+                <title>Announcements</title>
+                <meta name="Announcements" content="Description goes here." />
+            </Helmet>
+            <ChungusContentContainer>
+                <Content>
+                    <StyledTitle>Announcements</StyledTitle>
+                </Content>
+                <AnnouncementsList/>
```

Comment:
So, you'll have to do the connection to the redux store in this file, and dispatch the action here. Then, pass the resulting list in to the AnnouncementsList component as a prop of type `Announcement[]` (not AsyncRequest)